sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderAsyncClient.java

```diff
@@ -74,6 +75,9 @@
      */
     static final int MAX_MESSAGE_LENGTH_BYTES = 256 * 1024;
     private static final String TRANSACTION_LINK_NAME = "coordinator";
+    // Please see <a href=https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-services-resource-providers>here</a>
+    // for more information on Azure resource provider namespaces.
+    private static final String AZ_TRACING_NAMESPACE_VALUE = "Microsoft.ServiceBus";
```

Comment:
Do we have a ClientConstants in SB to put this?

Reply:
This can live here for now if it is the only place it is used.

Reply:
If you execute Ctrl+N, then Constants, you'll find ServiceBusConstants.